src/docfx.website.themes/default/styles/docfx.js

```diff
@@ -103,38 +100,43 @@ $(function () {
     });
   }
 
-  //Adjust the position of search box in navbar
-  function adjustSearchBoxPosition() {
-    autoCollapse();
-    $(window).on('resize', autoCollapse);
-    $(document).on('click', '.navbar-collapse.in', function (e) {
-      if ($(e.target).is('a')) {
-        $(this).collapse('hide');
-      }
-    });
-
-    function autoCollapse() {
-      var navbar = $('#autocollapse');
-      if (navbar.height() === null) {
-        setTimeout(autoCollapse, 300);
-      }
-      navbar.removeClass(collapsed);
-      if (navbar.height() > 60) {
-        navbar.addClass(collapsed);
-      }
-    }
-  }
-
   // Support full-text-search
-  function supportFullTextSearch() {
+  function enableSearch() {
     var query;
     var relHref = $("meta[property='docfx\\:rel']").attr("content");
-
-    if (typeof relHref != 'undefined') {
+    if (typeof relHref === 'undefined') {
+      return;
+    }
+    try {
       var search = searchFactory();
       search();
```

Comment:
searchFactory() returns a search worker, it would be searchFactory()()... looks like separating  is more readable.

---
In reply to: [108875653](https://github.com/dotnet/docfx/pull/1509#discussion_r108875653) [](ancestors = 108875653)